components/expenses/AdminExpenseStatusTag.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ChevronDown } from '@styled-icons/boxicons-regular/ChevronDown';
+import ReactDOM from 'react-dom';
+import { useIntl } from 'react-intl';
+import { Manager, Popper, Reference } from 'react-popper';
+import styled from 'styled-components';
+import { v4 as uuid } from 'uuid';
+
+import expenseStatus from '../../lib/constants/expense-status';
+import { i18nExpenseStatus } from '../../lib/i18n/expense';
+
+import { Box, Flex } from '../Grid';
+import StyledTag from '../StyledTag';
+
+import { getExpenseStatusMsgType } from './ExpenseStatusTag';
+import ProcessExpenseButtons from './ProcessExpenseButtons';
+
+const ExpenseStatusTag = styled(StyledTag)`
+  cursor: pointer;
+  font-weight: bold;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.06em;
+  text-transform: uppercase;
+`;
+
+const PopupContainer = styled(`div`)`
+  z-index: 2;
+  border-radius: 5px;
+  font-size: 12px;
+  text-transform: initial;
+  white-space: normal;
+  background: #ffffff;
+  box-shadow: 0px 4px 8px rgba(20, 20, 20, 0.16);
+`;
+
+const Arrow = styled('div')`
+  position: absolute;
+  font-size: 8px;
+  width: 3em;
+  height: 3em;
+  top: 0;
+  left: 0;
+  margin-top: -0.9em;
+  &::before {
+    content: '';
+    margin: auto;
+    display: block;
+    border-style: solid;
+    border-width: 0 1.5em 1em 1.5em;
+    border-color: transparent transparent #ffffff transparent;
+    filter: drop-shadow(0px -3px 3px rgba(20, 20, 20, 0.1));
+  }
+`;
+
+const ChevronDownIcon = styled(ChevronDown)`
+  width: 15px;
+  height: 15px;
+  cursor: pointer;
+  color: inherit;
+`;
+
+const AdminExpenseStatusTag = ({ expense, host, collective, ...props }) => {
+  const intl = useIntl();
+  const [showPopup, setShowPopup] = React.useState(false);
+  const [id, setId] = React.useState(false);
+
+  const onClick = () => {
+    setShowPopup(true);
+  };
+
+  const onMouseLeave = () => {
+    setShowPopup(false);
+  };
+
+  React.useEffect(() => {
+    setId(`popup-${uuid()}`);
+  });
+
+  const isMounted = Boolean(id);
+
+  return (
+    <React.Fragment>
+      <Manager>
+        <Reference>
+          {({ ref }) => (
+            <Box ref={ref} onClick={onClick} onMouseLeave={onMouseLeave}>
```

Comment:
The `onMouseLeave` behavior is a bit surprising, these are usually closed when users click - either inside or outside. See `useGlobalBlur`